### PR TITLE
fix(patterns/hero): code refactoring - update of the HTML structure

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [ ] No

## Describe the changes

Code refactoring + fix spacing issues

GitHub issue number or Jira issue URL: [ID-9456](https://adeo.kanbanize.com/ctrl_board/10/cards/9456/details/)

### CODE REVIEW

File: src/docs/Components/Hero/index.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ This following case with a boxed content in the hero can become hard to understa
 
 </Highlight>
 
-<Preview path="box" nude/>
+<Preview path="boxed-content" nude />
```
Comment (nherwegh) [inline src/docs/Components/Hero/index.mdx:50]: The box is under the text on the `medium` breakpoint

The cases are: 
- S & M ➡️ box under the text
- L ➡️ box under the text and on the right side of the text
- XL & XXL ➡️ box on the right side of the text

[Abstract collection](https://share.goabstract.com/293e5252-0102-4635-8188-2380ba6c7668?collectionLayerId=84c2c89e-3039-42e1-9e9c-b5a1818ae00d&mode=design)

**Review by nherwegh — CHANGES_REQUESTED** [review]

File: src/docs/Components/Hero/index.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/Components/Hero/index.mdx:50]: @nherwegh Normally this is already the case.
Can you confirm that this is good for you?

http://fix9456hero.demo.mozaic-design-system-dtrp.appspot.com/Components/Hero/#title-with-text-and-boxed-content

Comment (tiloyi) [reply src/docs/Components/Hero/index.mdx:50]: I just corrected it. 
I'll let you check it and tell me if it's good.

**Review by nherwegh — APPROVED** [review]
good for me 👍

**Review by mohamedMok — APPROVED** [review]

**Pull request merged**